utils/src/main/java/org/robolectric/util/TempDirectory.java

```diff
@@ -107,6 +107,7 @@ public void destroy() {
       clearDirectory(basePath);
       Files.delete(basePath);
     } catch (IOException ignored) {
+      Logger.error("IOException on clearing directory or deleting files", ignored);
```

Comment:
What about "Failed to destroy temp directory"?

Reply:
Sure, its more reasonable